api/machine/machine.proto

```diff
@@ -47,6 +48,19 @@ service MachineService {
   rpc Version(google.protobuf.Empty) returns (VersionResponse);
 }
 
+// rpc applyConfiguration
+// ApplyConfiguration describes a request to assert a new configuration upon a node.
+message ApplyConfigurationRequest {
+  bytes data = 1;
+
+  // enum op = 2; // (perhaps) patch, replace, etc
+
+  // bool no_reboot = 3; // (perhaps) do not reboot after application of new configuration
```

Comment:
Should we drop this idea into a ticket and remove the comments? Nitpicky, I know, but I fund that commented code is easy to lose.

Reply:
I really only put those there to support the idea of _having_ a request container, rather than as serious suggestions of features, actually.  I don't think they are _worthy_ of being made into issues, but I'm happy to remove the comments.  I think it's probably fairly self-evident what future possibilities there may be.

Reply:
Up to you, not a blocker for me.